src/Validation.Common.Job/Validation.Common.Job.csproj

```
@@ -17,6 +17,7 @@
     <Compile Remove="Validation\IValidationResponse.cs" />
     <Compile Remove="Validation\IValidator.cs" />
     <Compile Remove="SubscriptionProcessorJob.cs" />
+	<Compile Remove="Validation\ValidationResponse.cs" />
```

It looks like there's a tab here instead of whitespace:

```suggestion
    <Compile Remove="Validation\ValidationResponse.cs" />
```